.changeset/popular-plums-deny.md

```diff
@@ -2,4 +2,4 @@
 "@localyze-pluto/components": minor
 ---
 
-Add a Lucide Icons as dependency with a specific subset available
+Add a Lucide Icons as dependency with a specific subset available: baggage-claim, fingerprint, heart-handshake, palette, slack
```

Comment:
Won't we now have `fingerprint` from both Lucide and HeroIcons? Shouldn't we consolidate on 1 so that it's not inconsistently used?

Reply:
good call!